wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -147,17 +148,19 @@ body.sensei {
 				}
 			}
 
-			.wp-block-sensei-lms-course-outline-module__lessons-title {
-				display: none;
-			}
-
 			.wp-block-sensei-lms-collapsible {
+				overflow: unset;
```

Comment:
Changing `overflow: hidden` will only impact one scenario: when the window is resized on the desktop, causing text to overflow. However, in practical usage, when users initially access the page with different screen sizes, the height of the lesson items is automatically calculated, preventing any overflow. 

https://github.com/user-attachments/assets/0dd53e35-4feb-4e6f-8436-a74671f7d87d

If someone does this, in the case of an iPhone 5e, most titles will not overflow. Also, the originally hidden text (when `overflow: hidden`) cannot be seen through scrolling anyway, as this aspect is not specifically handled by Sensei.

![focus](https://github.com/user-attachments/assets/e9fe9903-00a9-4aec-a611-336f61042b11)